src/OrbitGl/TimeGraph.cpp

```diff
@@ -68,6 +68,7 @@ TimeGraph::TimeGraph(AccessibleInterfaceProvider* parent, OrbitApp* app,
   batcher_.SetPickingManager(picking_manager);
   track_container_ = std::make_unique<orbit_gl::TrackContainer>(
       this, this, viewport, &layout_, app, app->GetModuleManager(), capture_data);
+  timeline_ui_ = std::make_unique<orbit_gl::TimelineUi>(this, this, viewport, &layout_);
```

Comment:
Maybe add function argument comments, see [style guide entry](https://google.github.io/styleguide/cppguide.html#Function_Argument_Comments)

Reply:
I'm not sure if I got it, something like this:
`timeline_ui_ = std::make_unique<orbit_gl::TimelineUi>(/*parent*/this, /*timeline_info_interface*/this, viewport, &layout_);`

Reply:
yep that's it, thanks!

Reply:
Well with a "=" usually, like /*parent=*/